src/components/ChatApp/MessageComposer.react.js

```diff
@@ -164,6 +164,7 @@ class MessageComposer extends Component {
           Actions.createMessage(text, this.props.threadID);
         }
         this.setState({text: ''});
+        Button = <Mic />
```

Comment:
The same reason applies here, we need a check to confirm that the browser follows the Speech Recognition API first. If it doesn't, then the user will be stuck with a Mic infinitely.